docs/content/developers/design-keuzes.md

```diff
@@ -253,3 +253,63 @@ Een voorbeeld response is dan:
 ```
 
 Merk op dat de concrete codes hier fictief zijn en puur ter illustratie zijn.
+
+
```

Comment:
De PR van Henri legt ook nog uit hoe met 1-N relaties om te gaan.

Reply:
Over andere vormen van relaties gaat @joeribekker als het goed is even sparren vandaag op de Fieldlab :)

Reply:
Stukje verwerkt in deze PR. Ik had er eigenlijk geen bezwaren over, had gemist dat het over relaties binnen een component ging.